kyuubi-util-scala/src/test/scala/org/apache/kyuubi/util/AssertionUtils.scala

```diff
@@ -76,9 +76,8 @@ object AssertionUtils {
       path: Path,
       expectedLines: Traversable[String],
       regenScript: String,
-      splitFirstExpectedLine: Boolean = false)(implicit
-      prettifier: Prettifier,
-      pos: Position): Unit = {
+      splitFirstExpectedLine: Boolean =
+        false)(implicit prettifier: Prettifier, pos: Position): Unit = {
```

Comment:
too weird style ...

Reply:
Yes. Maybe we would find another way or config to get rid of it in the future.